workflow/serializers.py

```diff
@@ -52,29 +53,115 @@ class Meta:
             'label',
             'description',
             'widget_type',
-            'parameter_mappings',
+            'index',
+            'parameter_aliases',
             'section',
             'workflow',
-            'index'
         ]
 
 
-class WidgetParameterMapping(ModelSerializer):
+class ParameterAlias(ModelSerializer):
+
+    parameter_stub = ResourceRelatedField(
+        queryset=models.Parameter.objects.all(),
+        many=False,
+        required=False
+    )
+
+    parameter = SerializerMethodResourceRelatedField(
+        read_only=True,
+        source='get_parameter',
+        model=models.Parameter,
+        many=False,
+        required=False
+    )
+
+    def get_parameter(self, obj):
```

Comment:
Method to compute the related parameters. This, along with the the method on the parameter serializer for aliases, mean the frontend doesn't need to care about stubs.